bukkit/src/main/java/me/lokka30/treasury/plugin/bukkit/command/TreasuryCommand.java

```diff
@@ -0,0 +1,60 @@
+package me.lokka30.treasury.plugin.bukkit.command;
+
+import java.util.List;
+import me.lokka30.treasury.plugin.bukkit.Treasury;
+import me.lokka30.treasury.plugin.core.command.TreasuryBaseCommand;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.PluginCommand;
+import org.bukkit.command.TabExecutor;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class TreasuryCommand implements TabExecutor {
+
+    public static void register(Treasury plugin) {
+        PluginCommand cmd = plugin.getCommand("treasury");
+        CommandSources sources = new CommandSources();
+        plugin.getServer().getPluginManager().registerEvents(sources, plugin);
+        TreasuryCommand executor = new TreasuryCommand(sources);
+        cmd.setExecutor(executor);
+        cmd.setTabCompleter(executor);
+    }
+
+    private final TreasuryBaseCommand base;
+    private final CommandSources sources;
+
+    private TreasuryCommand(CommandSources sources) {
+        base = new TreasuryBaseCommand();
+        this.sources = sources;
+    }
+
+    @Override
+    public boolean onCommand(
+            @NotNull CommandSender sender,
+            @NotNull Command command,
+            @NotNull String label,
+            @NotNull String[] args
+    ) {
```

Comment:
What the hell happened with this formatting

Reply:
I tried to keep it as readable as possible.

Reply:
I think this should be merged into one line but it is a nitpick, it's really fine